code/modules/projectiles/projectile/bullets.dm

```diff
@@ -214,19 +218,19 @@
 			if(!isrobot(target))
 				L.apply_effect(5, DROWSY, 0)
 				if(def_zone == "torso")
-					if(blocked < 2 && !(blocked < 1))
+					if(blocked < 100 && !(blocked < 1))
 						target.visible_message("<b>[target]</b> yawns.")
-					if(blocked < 1)
+					if(blocked < 20)
 						spawn(120)

```

Comment:
Could you convert these three spawns to a timer please? Two are similar enough that they should be doable with a single proc that takes an argument.